web/src/components/Router/Router.tsx

```diff
@@ -1,18 +1,19 @@
+import Home from 'pages/Home';
+import RunDetail from 'pages/RunDetail';
+import Test from 'pages/Test';
+import Transaction from 'pages/Transaction';
 import {Navigate, Route, Routes} from 'react-router-dom';
 import {HistoryRouter} from 'redux-first-history/rr6';
 import {history} from 'redux/store';
 
-import Home from 'pages/Home';
-import Test from 'pages/Test';
-import RunDetail from 'pages/RunDetail';
-
 const {serverPathPrefix = '/'} = window.ENV || {};
 
 const Router = () => (
   <HistoryRouter history={history} basename={serverPathPrefix}>
     <Routes>
       <Route path="/" element={<Home />} />
       <Route path="/test/:testId" element={<Test />} />
+      <Route path="/transaction/:transactionId" element={<Transaction />} />
```

Comment:
This route should be something like `/transaction/:transactionId/run/:runId`.  And probably the `pages/Transaction` component should have a different name. If we compare this with tests:

`pages/Test` = `pages/Transaction`
`pages/RunDetail` = `pages/TransactionRunDetail ?` (this is the page you are implementing)

Reply:
✅